draft-ietf-quic-transport.md

```diff
@@ -2343,28 +2343,30 @@ source address.
 
 ## Idle Timeout {#idle-timeout}
 
-If the idle timeout is enabled, a connection is silently closed and the state is
-discarded when it remains idle for longer than both the advertised
-idle timeout (see {{transport-parameter-definitions}}) and three times the
+If the idle timeout is enabled by either peer, a connection is silently closed
+and its state is discarded when it remains idle for longer than the
+max_idle_timeout (see {{transport-parameter-definitions}}) and three times the
```

Comment:
This sentence seems to contradict the next paragraph: The connection is closed when it remains idle for longer than the minimum of the max_idle_timeouts.